app/controllers/admin/api/access_tokens_controller.rb

```diff
@@ -18,9 +18,10 @@ class Admin::Api::AccessTokensController < Admin::Api::BaseController
   ##~ e.responseClass = "access_token"
   #
   ##~ op = e.operations.add
+  ##~ op.deprecated = true
```

Comment:
I don't see any difference in the documentation... 😞 
![image](https://user-images.githubusercontent.com/11318903/51378445-96cc3180-1b0d-11e9-8bba-e707b38f22da.png)

Reply:
any difference in html? (so we can style)

Reply:
I see that we add (deprecated) to the desc in other places.

Reply:
Thanks Thomas, I will do the same then 😄 I am just waiting for more reviews first to do all the changes at once 👍